src/svg/ds6/icons.svg

```diff
@@ -84,6 +84,12 @@
     <symbol id="icon-help" viewBox="1.5 1.5 21 21.12">
         <path d="M17.48 22.623H2.297a.803.803 0 0 1-.798-.808c0-.446.357-.807.798-.807h13.585v-2.062c0-.949-.555-1.806-1.414-2.186l-2.616-1.104a1.96 1.96 0 0 1-1.198-1.864c.013-.602.29-1.155.76-1.517 1.03-.793 1.464-2.095 1.131-3.396a3.098 3.098 0 0 0-3.012-2.317c-.87 0-1.685.342-2.298.964a3.14 3.14 0 0 0-.914 2.223c.002 1 .455 1.92 1.242 2.526.47.362.747.915.76 1.517a1.96 1.96 0 0 1-1.197 1.864L4.51 16.76a2.373 2.373 0 0 0-1.41 2.043.8.8 0 0 1-.844.757.804.804 0 0 1-.749-.854 3.98 3.98 0 0 1 2.372-3.43l2.633-1.11a.354.354 0 0 0 .216-.337.348.348 0 0 0-.13-.268 4.783 4.783 0 0 1-1.872-3.808 4.756 4.756 0 0 1 1.38-3.368 4.78 4.78 0 0 1 3.427-1.439 4.697 4.697 0 0 1 4.558 3.528c.49 1.914-.181 3.91-1.71 5.087a.348.348 0 0 0-.13.268.353.353 0 0 0 .216.336l2.633 1.111a4.001 4.001 0 0 1 2.38 3.67v3.677zM16.79 4.09c.566-.472 1.115-.233 1.115-.233.822.775 1.088 1.604 1.11 2.513.022.958-.244 1.994-1.11 2.744 0 0-.572.15-1.115-.224-.542-.375-.29-.998-.29-.998.345-.419.561-.941.568-1.422.006-.5-.21-.956-.568-1.39 0 0-.276-.517.29-.99zm2.226-2.33c.492-.421 1.167-.206 1.167-.206 1.652 1.48 2.335 3.252 2.316 5.085-.02 1.793-.704 3.647-2.316 5.038 0 0-.67.322-1.167-.178-.496-.5-.196-1-.196-1 2.316-2.3 2.316-5.582 0-7.766 0 0-.295-.553.196-.974z"/>
     </symbol>
+    <symbol id="icon-pause" viewBox="0 0 16 16">
+        <g fill="currentColor" fill-rule="evenodd"><path d="M0 0h6v16H0zM10 0h6v16h-6z"></path></g>
```

Comment:
You don't need `<g>` at all, nor the other `fill` or `fill-rule`. It should simply be:

```
<path d="M0 0h6v16H0zM10 0h6v16h-6z"/>
```